core/templates/components/code-mirror/codemirror.component.ts

```diff
@@ -34,9 +34,9 @@ interface CodeMirrorMergeViewOptions {
 export class CodeMirrorComponent implements AfterViewInit, OnChanges {
   @Input() options!: CodeMirrorMergeViewOptions;
   // TODO(#13015): Remove use of unknown as a type.
-  // The property 'value' is dependent on other property 'localValue' from
-  // 'schema-based-editor'.Most part using it being currently in ajs it's
-  // type cannot be determined for now.
+  // The property 'value' is dependent on other property, 'localValue', from
+  // 'schema-based-editor'. Most components using 'localValue' are currently in
+  // AngularJS, so it's type cannot be determined for now.
```

Comment:
it's --> its

The former means "it is". The latter is the possessive pronoun.

Reply:
done